.travis.yml

```diff
@@ -65,6 +65,7 @@ before_install:
   - git clone --branch $FABRIC_VERSION https://github.com/hyperledger/fabric.git $FABRIC_PATH
   - pushd $FABRIC_PATH
   - git am $GOPATH/src/github.com/hyperledger-labs/fabric-private-chaincode/fabric/*.patch
+  - ln -s $GOPATH/src/github.com/hyperledger-labs/fabric-private-chaincode/tlcc $GOPATH/src/github.com/hyperledger/fabric/core/scc/tlcc
```

Comment:
so this must be done here, because tlcc is compiled into the peer?

Reply:
i guess. But the more problematic issue is that right now it doesn't work (see travis build): either the shell ignores the symlink and thinks it is in the fabric tree but then the relative paths to other fpc code fails or it knows about symlink the relative links to fpc work but then the whole reason to run it from fabric would be kindof moot and/or bring gopath problems?

Reply:
peer patching and compilation moved into the `fabric/Makefile`